.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
+
 node_js:
-  - "0.11"
-  - "0.10"
+    - 4.6
+    - 6.9
+
 notifications:
-  email: false
+    email: false
+
 before_install: npm install -g grunt-cli
-sudo: false
```

Comment:
It seems as if this build ran on Travis without sudo (https://travis-ci.org/boxuk/hubot-taphouse/jobs/181356385#L136), so should be correct.